src/Esfa.Vacancy.Register.Api/Constants/RouteNameConstants.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Esfa.Vacancy.Register.Api.Constants
+{
+    public class RouteNameConstants
```

Comment:
Constants.RouteNames would already be inferred from the namespace. so I'd be inclined to drop the "Constants" from the class name.

Reply:
Updated